README.md

```diff
@@ -74,6 +76,7 @@ appops set [package name] ACTIVATE_VPN allow
 | Candy Crush Saga | 1.213.2.1 (12132011) | 🟩 |||
 | Canvas Student | 6.14.1 | 🟩 |||
 | CarX Highway Racing | 1.17.1 | 🟨 | Terrible in-game experience, includes poor performance and low FPS ||
+| 超星学习通 (com.chaoxing.mobile) | 5.0.3 | 🟩 | Latest version crashes | Version 4.6.1 is fine |
```

Comment:
```suggestion
| 超星学习通 | 5.0.3 |🟨| Version (5.0.3) (latest) crashes on startup, but version 4.6.1 works ||
```

Reply:
If it crashes on startup, then this should be red and the note simply "Crashes on startup". A separate entry should be created for 4.6.1.

Reply:
> If it crashes on startup, then this should be red and the note simply "Crashes on startup". A separate entry should be created for 4.6.1.

looks good now?

Reply:
Perfect.